{"EventId":1915233606,"LogLevel":"Information","Category":"CustomCategory_TestEventNotificationSub","Message":"Handling event notification with payload\n{\n  \u0022PayloadSub\u0022: 11,\n  \u0022PayloadBase\u0022: 10\n}\n(Event Notification ID: test-message-id, Trace ID: test-trace-id)","State":{"Message":"Handling event notification with payload\n{\n  \u0022PayloadSub\u0022: 11,\n  \u0022PayloadBase\u0022: 10\n}\n(Event Notification ID: test-message-id, Trace ID: test-trace-id)","EventNotificationPayload":"{\n  \u0022PayloadSub\u0022: 11,\n  \u0022PayloadBase\u0022: 10\n}","EventNotificationId":"test-message-id","TraceId":"test-trace-id","{OriginalFormat}":"Handling event notification with payload\n{@EventNotificationPayload:l}\n(Event Notification ID: {EventNotificationId:l}, Trace ID: {TraceId:l})"}}
{"EventId":893113502,"LogLevel":"Error","Category":"CustomCategory_TestEventNotificationSub","Message":"An exception occurred while handling event notification after 123.4560ms (Event Notification ID: test-message-id, Trace ID: test-trace-id)","Exception":"Conqueror.Middleware.Logging.Tests.Eventing.LoggingMiddlewareTestEventNotifications\u002BTestException: test exception\n   at Conqueror.Middleware.Logging.Tests.Eventing.LoggingMiddlewareTestEventNotifications.TestEventNotificationBaseHandler.Handle(TestEventNotificationBase notification, CancellationToken cancellationToken) in {ProjectDirectory}Eventing/LoggingMiddlewareTestEventNotifications.cs:line 769\n   at Conqueror.Middleware.Logging.Eventing.LoggingEventNotificationMiddleware\u00601.Execute(EventNotificationMiddlewareContext\u00601 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Eventing/LoggingEventNotificationMiddleware.cs:line 51\n   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine\u0026 stateMachine)\n   at Conqueror.Middleware.Logging.Eventing.LoggingEventNotificationMiddleware\u00601.Execute(EventNotificationMiddlewareContext\u00601 ctx)\n   at Conqueror.Eventing.EventNotificationPipelineRunner\u00601.\u003C\u003Ec__DisplayClass3_2.\u003CExecute\u003Eb__1(TEventNotification notification, CancellationToken token) in {SolutionDirectory}src/Conqueror/Eventing/EventNotificationPipelineRunner.cs:line 36","State":{"Message":"An exception occurred while handling event notification after 123.4560ms (Event Notification ID: test-message-id, Trace ID: test-trace-id)","ResponseLatency":123.456,"EventNotificationId":"test-message-id","TraceId":"test-trace-id","{OriginalFormat}":"An exception occurred while handling event notification after {ResponseLatency:0.0000}ms (Event Notification ID: {EventNotificationId:l}, Trace ID: {TraceId:l})"}}
